.github/workflows/cd.yml

```diff
@@ -34,43 +34,59 @@ jobs:
         uses: mindsers/changelog-reader-action@v1
         with:
           version: ${{ steps.build.outputs.version }}
-      - name: Create GitHub release
+      - name: Create GitHub release and upload the distribution
         id: create-release
-        uses: actions/create-release@v1
```

Comment:
This is obsolete and was causing problems when run from the container. I use https://github.com/softprops/action-gh-release as in https://github.com/Backblaze/terraform-provider-b2